src/features/game/lib/constants.ts

```diff
@@ -496,7 +496,7 @@ export const INITIAL_FARM: GameState = {
     "Carrot Cake": new Decimal(1),
     Radish: new Decimal(100),
     Wheat: new Decimal(100),
-    Egg: new Decimal(15),
+    Egg: new Decimal(30),
```

Comment:
you could always discard all changes on the 'constant.ts' because it is all related to testing only.

Reply:
Yeah I see both ways on this, as we want to keep the main ‘constants.ts’ clean but also enable easy testing on branches. By keeping the constants file as one commit I think the devs can just not merge that commit if they choose

Reply:
Some tests might fail due to changes in constants.ts, be sure to run `yarn test` before committing.